test/test_backend/test_resolve_fetch_failures.py

```diff
@@ -45,18 +47,25 @@ class TestBackendResolveFetchFailures(FetchAuditTestRunner):
     mock_data = [{'table': 'code_sets', 'primary_key': 112182256, 'status_initially': 'fail-excessive-items',
                   'comment': 'Unit testing.'}]
 
+    def _failure_status_query(self, pk: Union[int, str]):
+        """Check status of fetch failure and its possible resolution"""
+        result = sql_query(self.con, f"SELECT success_datetime FROM fetch_audit WHERE primary_key = '{pk}';")
+        results2 = result[-1]
+        return results2
+
     # todo: A better test would be to actually run this in test_n3c, and check before/after that actual data is inserted
     def test_resolve_fetch_failures_excess_items(self):
         """Test resolve_fetch_failures_excess_items()"""
         pk = self.mock_data[0]['primary_key']
-        query = lambda: sql_query(self.con, f"SELECT success_datetime FROM fetch_audit WHERE primary_key = '{pk}';")[-1]
         # mock_data: setUpClass will have inserted by now
         if self.run_live_fetch:
             run_sql(self.con, f"DELETE FROM fetch_audit WHERE primary_key = '{pk}' AND comment = 'Unit testing.';")
             fetch_cset_and_member_objects(codeset_ids=[pk])
-        status1 = query()
-        resolve_fetch_failures_excess_items()
-        status2 = query()
+        status1 = self._failure_status_query(pk)
+        # todo: Ideally should be able to pass specific failures. What if actual failures exist that aren't test cases?
+        #  the it will run those as well.
+        resolve_fetch_failures_excess_items(TEST_SCHEMA, 24)
```

Comment:
## The main bug fixes (workaround solution)
I'm not actually sure what the source of these original errors were. One was happening during a `resolve_fetch_failures_excess_items()` assertion. It was checking on the status of a temporary mock fetch failure, but for some reason it got deleted early.

There was another but in `FetchAuditTestRunner`, where the number of rows was different at the end of the test. I inserted a todo that might address the issue (data being inserted by a live refresh while tests running), but I think this might be a rare case, so I don't know if it's a huge priority. I could be wrong about how rare that is, though, in which case I'll fix this sooner. Other than that though I'm not sure what could be causing the issue.  But the fact that it happened simultaneously with the `resolve_fetch_failures_excess_items()` issue above makes me think that they are related.

Workaround solution
I say workaround, but it's really a side effect of changing the tests to be set up in the way that they should have been. They were running off of `n3c`. The test was actually triggering a refresh of that schema, not `test_n3c`. After fixing this, the errors went away.